pom.xml

```diff
@@ -115,6 +115,14 @@
                 <plugin>
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>2.22.0</version>
+                    <configuration>
+                        <includes>
+                            <include>**/*Test.java</include>
+                        </includes>
+                        <excludes>
+                            <exclude>**/Main*.java</exclude>
+                        </excludes>
+                    </configuration>
```

Comment:
Is it necessary changes? Why do you need them?

Reply:
This requirement from checklist.md file - "We want to test only business logic, so there is no need to cover the Main class with tests. You can exclude the Main class with the main() method from being checked for code coverage in pom.xml". So, I have done it.